x-pack/plugin/core/src/main/config/log4j2.properties

```diff
@@ -3,7 +3,8 @@ appender.audit_rolling.name = audit_rolling
 appender.audit_rolling.fileName = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}_audit.json
 appender.audit_rolling.layout.type = PatternLayout
 appender.audit_rolling.layout.pattern = {\
-                "@timestamp":"%d{ISO8601}"\
+                "type":"audit", \
+                "timestamp":"%d{yyyy-MM-dd'T'HH:mm:ss,SSSZ}"\
```

Comment:
@albertzaharovits in ES server we decided to add timezone to timestamps in logs. I included this change to audit logs too, so these are consistent
discussion link:
https://github.com/elastic/elasticsearch/pull/36833#discussion_r244225243